extensions/wikia/SpecialHealthcheck/HealthController.class.php

```diff
@@ -0,0 +1,211 @@
+<?php
+
+/**
+ * Extra health checks controller.
+ *
+ * Currently supports requests for getting the status of database clusters
+ */
+class HealthController extends WikiaController {
+
+	private $current = '';
+
+	private $clusters = [ ];
```

Comment:
I won't it personally as I just used autoformatting and that's why I was left with.
What is your preferred way for the empty array?